archlinuxcn/sendanywhere/PKGBUILD

```diff
@@ -0,0 +1,59 @@
+# Maintainer: Yizheng Hu <huyizheng@hotmail.com>
+# Contributor: Xeonacid <h.dwwwwww@gmail.com>
+# Contributor: Ye Shu <shuye02@outlook.com>
+# Contributor: Aaron Fischer <mail@aaron-fischer.net>
+# Contributor: Ashwin Vishnu <ashuwish+arch at gmail dot com>
+
+pkgname=sendanywhere
+pkgver=20.11.6
+pkgrel=2
+arch=('x86_64')
+
+pkgdesc="Send Anywhere is a multi-platform file sharing service where users can directly share digital content in real time."
+url="http://send-anywhere.com"
+license=('custom')
+depends=('electron8')
+makedepends=('asar')
+
+source=("https://update.send-anywhere.com/linux_downloads/sendanywhere_latest_amd64.deb" "LICENSE")
+sha512sums=('1da80fa5f054dcb2a0d7717c1544baff884164d933d643dca52d1bda677bd989bb676f20e19db64f629d7d6324af430f192a52dad2868d6dad4ae978ed6871e7'
```

Comment:
I'd like to know if the checksum will be updated automatically? If not, should this be `SKIP`?

Reply:
It won't. `vcs_update` expects them to be `SKIP`.

Reply:
Thanks! I've opened a PR #2159 to fix it.